AmplitudeServiceKitUI/AmplitudeServiceTableViewController.swift

```diff
@@ -0,0 +1,147 @@
+//
+//  AmplitudeServiceTableViewController.swift
+//  AmplitudeServiceKitUI
+//
+//  Created by Darin Krauss on 6/20/19.
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
+//
+
+import UIKit
+import LoopKit
+import LoopKitUI
+import AmplitudeServiceKit
+
+
+final class AmplitudeServiceTableViewController: ServiceTableViewController, UITextFieldDelegate {
+
+    private let amplitudeService: AmplitudeService
+
+    init(amplitudeService: AmplitudeService, for operation: Operation) {
+        self.amplitudeService = amplitudeService
+
+        super.init(service: amplitudeService, for: operation)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.register(AuthenticationTableViewCell.nib(), forCellReuseIdentifier: AuthenticationTableViewCell.className)
```

Comment:
Just want to flag the `.nib()` since I recently saw a change to `.self`.

Reply:
The use of `.self` or `.nib()` depends upon whether the view is implemented strictly in code (where `.self` would be used) or whether it uses a `.xib` file (where `.nib()` would be used). 

In the case of the recent change from `.nib()` to `.self` for `SwitchTableViewCell`, the `.xib` file was recently deleted and the view implemented only in code. See 9b23b8330f3507aa39ccd3f716b556b033f3c8ba for details.

In the above case, the `AuthenticationTableViewCell` uses a `.xib` file so the use of `.nib()` is correct.